workflow-templates/im-deploy-on-prem-website.yml

```
@@ -0,0 +1,462 @@
+# Workflow Code: FuzzyDragon_v1    DO NOT REMOVE
+
+# Prerequisites
+# 1 - Make sure the Service account secrets have been added to the environment
+# 2 - Make sure the host IIS server has been prepped to accept incoming WinRM Requests
+#     See the IIS App Pool action README for more details: https://github.com/im-open/iis-service-action/#readme
+# 3 - If you're using a deployment board, create it first: https://github.com/im-practices/git-er-done/blob/main/actions/deployment-board.md
+
+name: Deploy On Prem Website
+on:
+  workflow_dispatch:
+    inputs:
+      branch-tag-sha:
+        description: The branch, tag or sha of the code that should be deployed.
+        required: true
```

Pre-built artifact change:
```suggestion
      tag:
        description: The repository tag pointing to the code that should be deployed.  For Production, only tags reachable by the default branch will be accepted.
        required: true
```